src/main/resources/db/migration/V13_4__personal_officer_history.sql

```diff
@@ -0,0 +1,88 @@
+create or replace function initialise_po_record(allocation_id uuid) returns void as
+$$
+declare
+    _rev_id                         bigint;
+    _allocated_at                   timestamp;
+    _allocated_by                   varchar;
+    _deallocation_at                timestamp;
+    _deallocated_by                 varchar;
+    _deallocated                    boolean;
+begin
+
+    select allocated_at,
+           allocated_by,
+           deallocated_at,
+           deallocated_by,
+           not is_active
+    into _allocated_at, _allocated_by, _deallocation_at, _deallocated_by, _deallocated
+    from allocation
+    where id = allocation_id;
+
+    _rev_id = nextval('audit_revision_id_seq');
+    insert into audit_revision(id, timestamp, username, caseload_id, affected_entities)
```

Comment:
I'm assuming this will be run after clearing the existing audit history? Worth adding a check for existing rows or a delete of those rows anyway to stop duplicate rows?